.NET/Microsoft.Recognizers.Text.DateTime/Arabic/Extractors/ArabicDateTimeExtractorConfiguration.cs

```diff
@@ -85,6 +85,7 @@ public ArabicDateTimeExtractorConfiguration(IDateTimeOptionsConfiguration config
             TimePointExtractor = new BaseTimeExtractor(new ArabicTimeExtractorConfiguration(this));
             DurationExtractor = new BaseDurationExtractor(new ArabicDurationExtractorConfiguration(this));
             UtilityConfiguration = new ArabicDatetimeUtilityConfiguration();
+            HolidayPointExtractor = new BaseHolidayExtractor(new ArabicHolidayExtractorConfiguration(this));
```

Comment:
What's the difference between HolidayPointExtractor, SingleHolidayExtractor, and HolidayExtractor?
Shouldn't all be the same name and any individual differences just be handled based on config? That's what it looks like as they're always BaseHolidayExtraction created with a language-specific config.

Reply:
renamed  HolidayPointExtractor, SingleHolidayExtractor to HolidayExtractor.